test/Interop/Cxx/class/Inputs/constructors.h

```diff
@@ -0,0 +1,39 @@
+struct ExplicitDefaultConstructor {
+  ExplicitDefaultConstructor() : x(42) {}
+  int x;
+};
+
+struct ImplicitDefaultConstructor {
```

Comment:
We should also have a non-trivial version of `ImplicitDefaultConstructor`. See [the issue in #32591](https://github.com/apple/swift/pull/32591#issuecomment-650841400).

Reply:
`StructWithCopyConstructorAndValue` accomplishes this.